src/utils/utils.css

```diff
@@ -0,0 +1,12 @@
+[inline] {
+	display: inline;
+}
+
+[inline-block] {
+	display: inline-block;
+}
+
+/* Change the inconsistent appearance in IE (opinionated) */
```

Comment:
At this point if anyone is still using IE it may be a security vulnerability. xD

Reply:
Lol hahah